x-pack/plugins/observability_solution/observability/server/lib/rules/custom_threshold/lib/create_custom_metrics_aggregations.ts

```diff
@@ -59,6 +63,19 @@ export const createCustomMetricsAggregations = (
       };
     }
 
+    if (aggregation === Aggregators.LAST_VALUE) {
+      bucketsPath[metric.name] = key;
+      return metric.field
+        ? {
+            ...acc,
+            ...createLastValueAggBucket(key, timeFieldName, metric.field),
+            ...createLastValueAggBucketScript(key, metric.field),
```

Comment:
nit: if these two aggregations need to always be used together, maybe we can use only one function for this purpose.